app/exporters/publishing_api_draft_section_exporter.rb

```diff
@@ -1,6 +1,6 @@
 class PublishingApiDraftSectionExporter
   def call(section, manual)
-    organisation = OrganisationFetcher.fetch(manual.organisation_slug)
+    organisation = OrganisationsAdapter.new.find(manual.organisation_slug)
```

Comment:
Possibly, but I think I'd prefer to introduce some kind of "global registry" where we can obtain *instances* of these adapters, c.f. the `Services` module. However, I think I'll do that as a separate PR or possibly as part of another PR where having it would make life easier.